gluten-flink/docs/Flink.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: Gluten For Flink with Velox Backend
+nav_order: 1
+---
+
+# Supported Version
+
+| Type  | Version                      |
+|-------|------------------------------|
+| Flink | 1.19.2                       |
+| OS    | Ubuntu20.04/22.04, Centos7/8 |
+| jdk   | openjdk11/jdk17              |
+| scala | 2.12                         |
+
+# Prerequisite
+
+Currently, with static build Gluten+Flink+Velox backend supports all the Linux OSes, but is only tested on **Ubuntu20.04**. With dynamic build, Gluten+Velox backend support **Ubuntu20.04/Ubuntu22.04/Centos7/Centos8** and their variants.
+
+Currently, the officially supported Flink version is 1.19.2.
+
+We need to set up the `JAVA_HOME` env. Currently, Gluten supports **java 11** and **java 17**.
+
+**For x86_64**
+
+```bash
+## make sure jdk11 is used
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+export PATH=$JAVA_HOME/bin:$PATH
+```
+
+**For aarch64**
+
+```bash
+## make sure jdk11 is used
+export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-arm64
+export PATH=$JAVA_HOME/bin:$PATH
+```
+
+**Get Velox4j**
+
+Gluten for Flink depends on [Velox4j](https://github.com/velox4j/velox4j) to call velox. This is an experimental feature.
+You need to get the Velox4j code, and compile it first.
```

Comment:
@shuai-xu Can we provide a fixed tag or branch of Velox4J we are relying on here?